setup.py

```diff
@@ -54,7 +54,7 @@ def fix_exts(sources):
 
 setup(
     name='fastparquet',
-    version='0.1.6',
+    version='0.1.6.feature.optional_columns_to_float.0',
```

Comment:
There is no need to do this, I will take care of the version number for release after the PR is merged.
Most of my projects have moved to versioneer, which automatically adds git information to the version number, but not this one.

Reply:
sorry, did that for my internal build pipeline; need to remove it?

Reply:
Yes, I think its best if the version number only changes at times of release.

Reply:
want me to re-write git history or just remove it?

Reply:
Just remove is fine.

Reply:
done, need to check later why the tests fail